core/ir/opnd_api.h

```diff
@@ -2153,6 +2164,26 @@ opnd_t
 opnd_create_base_disp(reg_id_t base_reg, reg_id_t index_reg, int scale, int disp,
                       opnd_size_t data_size);
 
+#ifdef RISCV64
+DR_API
+/**
+ * Returns a memory reference operand that refers to the address:
+ * - disp(base_reg, index_reg, scale)
+ *
+ * or, in other words,
+ * - base_reg + index_reg*scale + disp
+ *
+ * The operand has data size data_size (must be a OPSZ_ constant).
+ * Both \p base_reg and \p index_reg must be DR_REG_ constants.
+ * \p scale must be either 0, 1, 2, 4, or 8.
+ *
+ * The integer will be formatted as a decimal when disassemble.
+ */
+opnd_t
+opnd_create_base_disp_decimal(reg_id_t base_reg, reg_id_t index_reg, int scale, int disp,
```

Comment:
Let's remove this too and use opnd_add_flags with the existing creation routine.